pages/community/precice-workshop-2022.md

```diff
@@ -19,12 +19,197 @@ If you have any questions regarding the workshop, please contact us at [`precice
+   </details>
+
+   <details class="workshop-event" id="Chourdakis"><summary>Gerasimos Chourdakis: Testing the multi‐component preCICE ecosystem </summary>
+   <p>Authors: <a>Gerasimos Chourdakis</a> <br/></p>
+   <p>With several bindings, adapters, tutorials, and more components now in its arsenal, preCICE is now much more than a coupling library: it is a rapidly growing multiphysics ecosystem. One small code contribution in any of the involved repositories can have side-effects on the building, running, and computations of any downstream component. Creating a sustainable testing framework for such a complex ecosystem is not trivial. This talk will discuss the status quo of testing complete coupled simulations for regressions, the ideal system for all the involved stakeholders, challenges specific to preCICE, and novel solutions that will lead us to the new preCICE system tests.</p>
+   </details>
+
+    <details class="workshop-event" id="Rodenberg"><summary>Benjamin Rodenberg: From low-order to high-order coupling schemes </summary>
+    <p>Authors: <a>Benjamin Rodenberg</a> <br/></p>
+    <p>preCICE offers explicit and implicit coupling schemes. They often can only reach first-order accuracy in time. We currently develop an extended coupling scheme that allows time interpolation of coupling data. With this one can generally reach higher order. In this talk I give a practical introduction for low-order and high-order coupling schemes in preCICE.  </p>
+    </details>
+
+   <details class="workshop-event" id="Schneider"><summary>David Schneider: Data Mapping in preCICE</summary>
+   <p>Authors: <a>David Schneider</a> <br/></p>
+   <p>This talk provides an overview of current mesh mapping methods in preCICE. A more detailed explanation of the radial basis function implementation is provided, along with some mesh mapping test results. Throughout the talk, best practice methods are suggested, and rounded off with future plans to enable easier mapping in preCICE.</p>
+   </details>
+
+* 12:00 - 12:30 Speakers in breakout rooms for Q&A
+* 12:30 - 13:30 Break (Offline)
+* 13:30 - 15:00 preCICE Course III (Main Hall)
+
+  <details class="workshop-event" id="courseIII"><summary>preCICE Course III: </summary>
+  <p>Instructors: <a href="https://www.in.tum.de/en/i05/people/personen/gerasimos-chourdakis/">Gerasimos Chourdakis</a>, <a href="https://www.simtech.uni-stuttgart.de/exc/people/Uekermann/">Benjamin Uekermann</a><br/>
+  Affiliation: Technical University of Munich, University of Stuttgart, preCICE developers.</p>
+  </details>
```

Comment:
@MakisH @uekerman I need some input on how you want to split the course content here in order to provide a proper summary in the program.